src/ui/router/ApplicationRouter.js

```diff
@@ -136,6 +136,13 @@ class ApplicationRouter extends EventEmitter {
         this.handleLocationChange(hash.substring(1));
     }
 
+    isNavigatedObject(objectPath) {
```

Comment:
This is in another PR which I am just about to merge - https://github.com/nasa/openmct/pull/4179/files#diff-0bff5a6f99db64e9b5322caa61d2aa0344eda15148a8fba7ed4bcc992d5254f6R39

Reply:
We shall see which one wins ;)